java/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more 
+  ~ contributor license agreements. See the NOTICE file distributed with 
+  ~ this work for additional information regarding copyright ownership. 
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0 
+  ~ (the "License"); you may not use this file except in compliance with 
+  ~ the License. You may obtain a copy of the License at 
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0 
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software 
+  ~ distributed under the License is distributed on an "AS IS" BASIS, 
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+  ~ See the License for the specific language governing permissions and 
+  ~ limitations under the License. 
+  -->
+  
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  
+  <parent>
+    <artifactId>interpreter-parent</artifactId>
+    <groupId>org.apache.zeppelin</groupId>
+    <version>0.9.0-SNAPSHOT</version>
+    <relativePath>../interpreter-parent</relativePath>
+  </parent>
+  
+  <groupId>org.apache.zeppelin</groupId>
+  <artifactId>zeppelin-java</artifactId>
+  <packaging>jar</packaging>
+  <version>0.9.0-SNAPSHOT</version>
+  <name>Zeppelin: Java interpreter</name>
+  
+  <properties>
+    <!--library versions-->
+    <interpreter.name>java</interpreter.name>
+  </properties>
+  
+  <dependencies>
+
+    <dependency>
+      <groupId>com.thoughtworks.qdox</groupId>
+      <artifactId>qdox</artifactId>
+      <version>2.0-M3</version>
```

Comment:
LICENSE update?

Reply:
ok I saw the PR comment on this

Reply:
Yes, it was already updated as part of the Beam interpreter.